core/unit_test/CMakeLists.txt

```diff
@@ -3,19 +3,10 @@ if(NOT Cabana_ENABLE_TESTING)
   return()
 endif()
 
-set(GTEST_SOURCE_DIR ${CMAKE_SOURCE_DIR}/gtest)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGTEST_HAS_PTHREAD=0")
```

Comment:
Do we not need to set `GTEST_HAS_PTHREAD` any more?

Reply:
I don't think so - that was a holdover from a failing Mac build which is working now

Reply:
I don't think so - that was a holdover from a failing Mac build which is working now